crypto/s390x_arch.h

```diff
@@ -77,6 +79,13 @@ __attribute__ ((visibility("hidden")))
 #endif
 extern struct OPENSSL_s390xcap_st OPENSSL_s390xcap_P;
 
+#ifdef S390X_MOD_EXP
+#  if defined(__GNUC__) && defined(__linux)
```

Comment:
style nit: single space after # not two.

Reply:
fixed